dbmongo/lib/marshal/parser.go

```diff
@@ -93,13 +93,15 @@ func runParserWithSirenFilter(parser Parser, filter *SirenFilter, filePath strin
 		for lineResult := range parsedLineChan {
 			filterError := lineResult.FilterError
 			if filterError != nil {
-				tracker.AddFilterError(filterError)
+				tracker.AddFilterError(filterError) // on rapporte le filtrage, même aucun tuple n'est transmis
```

Comment:
```suggestion
				tracker.AddFilterError(filterError) // on rapporte le filtrage, mais aucun tuple n'est transmis
```

Reply:
En fait, c'est ça que je voulais dire:

```suggestion
				tracker.AddFilterError(filterError) // on rapporte le filtrage même si aucun tuple n'est transmis par le parseur
```